html-block-and-inline/html-block-and-inline-quiz-notes.md

```diff
@@ -0,0 +1,49 @@
+# html-block-and-inline-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- How do block-level elements affect the document flow?
+
+Starts on a new line
+
+- How do inline elements affect the document flow?
+
+Do not start on a new line
```

Comment:
nline elements only take up as much width as the element takes. It will appear next to the previous element